charts/cosigned/README.md

```diff
@@ -7,18 +7,13 @@
 ### Deploy `cosigned` Helm Chart
 
 
+Let's create a secret to validate the signatures, once we define `k8s://` scheme for the command, cosign will generate public/private key pairs, and store them in a Secret without touching a disk:
 ```shell
 export COSIGN_PASSWORD=<my_cosign_password>
-cosign generate-key-pair
-```
-
-The previous command generates two key files `cosign.key` and `cosign.pub`. Next, create a secret to validate the signatures:
 
-```shell
 kubectl create namespace cosign-system
 
-kubectl create secret generic mysecret -n \
-cosign-system --from-file=cosign.pub=./cosign.pub
+cosign generate-key-pair k8s://cosign-system/mysecret
```

Comment:
in the secret, we just need to pub key afaik, not sure if we want to have the private key together.
thoughts @hectorj2f ?

Reply:
Yes, I won't suggest adding both for security reasons. That is the intention of only adding the public key.

Reply:
It could also mislead users which simply copy and paste these instructions, and come back asking why we need the private key.

Reply:
In the current design in cosign, we store both passwords, private and public keys in Kubernetes Secret btw.

Reply:
> In the current design in cosign, we store both passwords, private and public keys in Kubernetes Secret btw.

Yes. But cosigned doesn't need a private key to work.

Reply:
There was actually some confusion around this in slack a few weeks ago - it's best to keep the private key out of here!